tests/test_integration.py

```diff
@@ -896,3 +898,65 @@ def test_long_path(tmp_path: pathlib.Path, tmp_run: _T_TmpRun) -> None:
 
     assert re.search(r"venv_py[0-9]+_requests", result.stderr) is None
     assert "TEST OK" in result.stdout
+
+
+def test_requirements_lockfile_created(
+    tmp_path: pathlib.Path, tmp_run: _T_TmpRun
+) -> None:
+    rf_path = tmp_path / "riotfile.py"
+    rf_path.write_text(
+        """
+from riot import Venv
+venv = Venv(
+    pys=[3],
+    pkgs={"requests": ""},
+    name="test",
+    command="env",
+    venvs=[Venv(pkgs={"pytest": ""})]
+)
+""",
+    )
+    _hash = tmp_run("riot list --hash-only").stdout.strip("\n")
+    result = tmp_run("riot requirements {}".format(_hash))
+    assert result.returncode == 0, result.stderr
+
+    lockfile_path = str(
+        tmp_path
+        / os.path.join(
+            ".riot",
+            "requirements",
+            "{}.txt".format(_hash),
+        )
+    )
+    assert os.path.exists(lockfile_path)
```

Comment:
do we want to validate the contents?

we should trust `pip-tools`, but it should also be following a fairly expected format.

(we might need to pin specific pkg versions to get consistent behavior though, unless we just validate `requests==.*?$`)